Base/Systems/FullScreenSpriteSystem.cs

```diff
@@ -14,11 +14,12 @@ private partial struct RecalculateSpritesJob : IJobEntity
             public float2 CameraPosition;
             public float2 ScreenSize;
             
-            private void Execute(ref Scale2D size, ref WorldPosition2D position, ref UVTilingAndOffset uvTilingAndOffset, in NativeSpriteSize nativeSpriteSize)
+            private void Execute(ref Scale2D scale, ref LocalTransform2D transform, ref UVTilingAndOffset uvTilingAndOffset, in NativeSpriteSize nativeSpriteSize)
             {
-                size.value = ScreenSize;
-                position.value = CameraPosition;
-                uvTilingAndOffset.value = new float4(size.value / nativeSpriteSize.Value, CameraPosition / nativeSpriteSize.Value - size.value / nativeSpriteSize.Value / 2f);
+                scale.value = ScreenSize;
+                transform.Scale = 1;
+                transform.Position = CameraPosition;
+                uvTilingAndOffset.value = new float4(ScreenSize / nativeSpriteSize.Value, CameraPosition / nativeSpriteSize.Value - ScreenSize / nativeSpriteSize.Value / 2f);
```

Comment:
Why here we have `ref Scale2D` which is set to `ScreenSize` and at the same time we have `ref LocalTransform2D` which `Scale` is set to 1.

Reply:
Not sure why I choose to do it this way, probably to absolutely making sure it doesn't render unneccesary sprites

Reply:
> Not sure why I choose to do it this way, probably to absolutely making sure it doesn't render unneccesary sprites

How you construct your `LocalToWorld2D`. Is there TRS components or just one matrix with direct editing through properties? If so then we don't need `Scale2D` as a separate component, I guess

Reply:
The main reason I kept Scale2D is because for my project, I multiply local points by the LTW2D matrix, but don't want them to be affected by the scale of the sprite, which might be different

Reply:
> The main reason I kept Scale2D is because for my project, I multiply local points by the LTW2D matrix, but don't want them to be affected by the scale of the sprite, which might be different

Ok, so I will exclude extra `Scale2D` or will add TRS components, because I believe package should deliver minimal implimentation